pkg/stash/stasher_test.go

```diff
@@ -90,9 +91,12 @@ func (ms *mockStorage) Save(ctx context.Context, module, version string, mod []b
 	return nil
 }
 
-func (ms *mockStorage) Exists(ctx context.Context, mod, ver string) (bool, error) {
+func (ms *mockStorage) Info(ctx context.Context, mod, ver string) ([]byte, error) {
 	ms.existsCalled = true
-	return ms.existsResponse, nil
+	if ms.existsResponse {
+		return []byte{'e', 'x', 'i', 's', 't', 's'}, nil
+	}
```

Comment:
can we change these to `infoCalled` and `infoResponse` respectively? seems important to change the names to indicate that `Info` was called to test existence